### PR TITLE
Issue #3273994 by agami4: Improve font type

### PR DESCRIPTION
**Pull request by agami4**
## Problem
1. Default font color is changed to #343434 everywhere on the platform instead of current #4D4D4D
2. Line height for 16px font size is changed from 24px to 26px everywhere on the site
3. All fonts are regular (and not light)
4. Changes should be applied in Theme and Extensions

## Solution
Improve font type according to acceptance criteria.

## Issue tracker
https://www.drupal.org/project/social/issues/3273994
https://getopensocial.atlassian.net/browse/YANG-7372

## Theme issue tracker
https://www.drupal.org/project/socialblue/issues/3273918 ([MR](https://git.drupalcode.org/project/socialblue/-/merge_requests/74))
https://www.drupal.org/project/socialbase/issues/3273919 ([MR](https://git.drupalcode.org/project/socialbase/-/merge_requests/116))

## How to test
*[Required] For example*
- [ ] You should check pages.

## Screenshots
before:
Screenshot 2022-05-10 at 12 00 31

after:
Screenshot 2022-05-10 at 12 03 02

## Release notes
Improve font type:
  1. Default font color is changed from #4D4D4D to #343434
  2. Line height for 16px font size is changed from 24px to 26px
  3. All fonts are regular (and not light)

### CODE REVIEW

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2865-8cbpao3xb5icahivkbipdz7tk5nhpqu7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/624ee1017a1d6289544dc8f3

**Comment by Ressinel** [comment]
Hi @agami4 

Could you prepare this PR fully to review since you added the label `needs review`?
Do I need to be assigned to this PR?

**Review by Ressinel — CHANGES_REQUESTED** [review]

File: modules/social_features/social_core/social_core.install [hunk 1]
```diff
@@ -1547,3 +1547,16 @@ function social_core_update_11006(): void {
   user_role_grant_permissions('contentmanager', ['access social_core dashboard']);
   user_role_grant_permissions('sitemanager', ['access social_core dashboard']);
 }
+
+/**
+ * Update Social Blue theme with new accent text color.
+ */
+function social_core_update_11007(): void {
```
[the remark this answers is not in the document]
Comment (agami4) [reply modules/social_features/social_core/social_core.install]: @Ressinel, done. Please check

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2865-8cbpao3xb5icahivkbipdz7tk5nhpqu7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/624ee1017a1d6289544dc8f3

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2865-8cbpao3xb5icahivkbipdz7tk5nhpqu7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/624ee1017a1d6289544dc8f3

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2865-8cbpao3xb5icahivkbipdz7tk5nhpqu7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/624ee1017a1d6289544dc8f3

**Review by ribel — APPROVED** [review]
Hi @agami4 and @Ressinel,
What is the status here?
For me, it looks like completed, but I think we will need to merge related theme changes to the 2.x branch and update the theme version here?

**Review by Ressinel — APPROVED** [review]

**Comment by Ressinel** [comment]
@ribel Looks good to me as well.
We will merge related themes PRs to new dev branches 2.2.x and those theme changes will release with these changes in the distro. We will update the distro theme version right before the new distro release.

**Comment by Ressinel** [comment]
@agami4 Could you remove redundant commits? And we will be ready to merge. Thanks!

**Comment by tbsiqueira** [comment]
Hi @agami4, can you please add before and after screenshots with detailed information on what changed? This will be very important to add on our release notes. 

I would like to kindly as you to include the screenshots with all changes on this PR and d.org related ticket.

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2865-8cbpao3xb5icahivkbipdz7tk5nhpqu7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/624ee1017a1d6289544dc8f3

**Comment by agami4** [comment]
@Ressinel, @tbsiqueira, I've added screenshots(before, after) on the description. Please check.

**Comment by Ressinel** [comment]
🍒-picked to `11.3.x`

**Pull request merged**